a3m/client/clientScripts/change_object_names.py

```diff
@@ -31,19 +31,19 @@
 logger = logging.getLogger(__name__)
 
 
-class NameSanitizer:
+class NameChanger:
     """
-    Class to track batch sanitizations of files and directories, both in the
+    Class to track batch changes of files and directories, both in the
```

Comment:
I think this should specify that the change is a name change, e.g. "Class to track batch changes of file and directory names..."